src/main/java/org/myrobotlab/service/Joystick.java

```diff
@@ -177,12 +177,14 @@ public void poll() {
         if (pollingController == null) {
           error("controller is null - can not poll");
           stopPolling();
+          break;

```

Comment:
does this kill the polling thread?  is this change related to tracking?

Reply:
10K ways of implementing a worker ..
it was one of my early implementations ...  it seems kind of goofy ... i should refactor it into my most recent standards ...
but yes the thread will leave run